src/plugins/metric_vis/public/metric_vis_controller.js

```diff
@@ -8,13 +9,22 @@ define(function (require) {
 
     var metrics = $scope.metrics = [];
 
+    $scope.isNullorNaN = function (val) {
+      return _.isNull(val) || isNaN(val);
```

Comment:
Can you change this to `_.isNaN(val)`? The global `isNaN` is not a good part of JS, it has weird behavior for non-numeric arguments.